spec/dummy/db/schema.rb

```diff
@@ -11,13 +11,13 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20150727145934) do
+ActiveRecord::Schema.define(version: 20150727151245) do
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
   create_table "event_capture_events", force: :cascade do |t|
-    t.datetime "emitted_at"
+    t.datetime "emitted_at",  null: false
```

Comment:
Will this potentially break the staging or production environments where there may be "emitted_at" and "recorded_at" values of null?